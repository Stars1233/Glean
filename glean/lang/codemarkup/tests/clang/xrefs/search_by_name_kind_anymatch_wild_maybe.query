query: search.code.SearchByNameKindAndLanguage { name = _, language = Cpp, kinds = { just = _ } }
perf: true
transform: [gensort, []]
