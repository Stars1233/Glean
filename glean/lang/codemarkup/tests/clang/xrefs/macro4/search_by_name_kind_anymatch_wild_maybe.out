[
  "@generated",
  {
    "key": {
      "name": "N",
      "language": 0,
      "kinds": 5,
      "entity": {
        "cxx": {
          "defn": {
            "namespace_": {
              "key": {
                "declaration": {
                  "key": {
                    "name": { "key": { "name": { "key": "N" } } },
                    "source": {
                      "file": { "key": "test.cpp" },
                      "lineBegin": 11,
                      "columnBegin": 1,
                      "lineEnd": 13,
                      "columnEnd": 1
                    }
                  }
                },
                "members": {
                  "key": [
                    {
                      "function_": {
                        "key": {
                          "name": {
                            "key": {
                              "name": { "key": { "name": { "key": "foo" } } },
                              "scope": {
                                "namespace_": {
                                  "key": { "name": { "key": "N" } }
                                }
                              }
                            }
                          },
                          "signature": {
                            "key": {
                              "returns": { "key": "void" },
                              "parameters": [ ]
                            }
                          },
                          "source": {
                            "file": { "key": "test.cpp" },
                            "lineBegin": 12,
                            "columnBegin": 3,
                            "lineEnd": 12,
                            "columnEnd": 15
                          }
                        }
                      }
                    }
                  ]
                }
              }
            }
          }
        }
      }
    }
  }
]